### ISSUE TITLE
[COGNITION]: Data SHOULD not be ALL-CAPS

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#10511]
#Description
There are a few data points in the GIBCT that use all caps. Text in all capital letters is more difficult to read for most people, with and without disabilities. which is difficult for individuals with dyslexia to read.  Readability is reduced with all caps because all words have a uniform rectangular shape, meaning readers can't identify words by their shape. People with dyslexia would especially beneift from sentence or title casing where all caps currently appear.

I noticed all caps in a few places:
- Institution name, in the type-ahead, and in the cards
- City, in the cards
- Institution type dropdown options
- Country drop down

Based on similar issues in the repo, I think this is a [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Point of Contact

I'm (Liz Lantz) opening this issue simply because it's something I noticed...not sure how helpful I will be as the point of contact. But here I am :)

## Acceptance Criteria

Data points listed above use sentence or title casing as appropriate, instead of all caps.

## Environment

Mac OS, Chrome

## Steps to Recreate

[Visit the GIBCT](https://www.va.gov/gi-bill-comparison-tool/) and view the fields listed above.

## Possible Fixes (optional)

Is it possible to programmatically change to sentence case or title case? I'm pretty sure title case would be appropriate for all of these fields.  Perhaps someone can write a `data.toTitleCase()`; function or something?

## Screenshots or Trace Logs
<img width="256" alt="Screen Shot 2020-06-22 at 7 05 48 AM" src="https://user-images.githubusercontent.com/55992116/85281197-913bef00-b457-11ea-8497-a540b7c8852f.png">
<img width="277" alt="Screen Shot 2020-06-22 at 7 05 30 AM" src="https://user-images.githubusercontent.com/55992116/85281200-926d1c00-b457-11ea-9b19-2f5b8e0bc9bf.png">
<img width="704" alt="Screen Shot 2020-06-22 at 7 04 59 AM" src="https://user-images.githubusercontent.com/55992116/85281206-939e4900-b457-11ea-9d0b-cef1cff8c3c8.png">

**Comment by andaleliz** [comment]
@jenstrickland, when you get a moment, would you please give this a once-over? I want to make sure I've captured things thoroughly/accurately. Also, I'd like to confirm that i don't need to assign it to a person, given the labels that it has.  Thanks!

**Comment by jenstrickland** [comment]
@andaleliz Thanks! I suspect this is an issue with the database information, and the team is likely aware of it. I tagged @1Copenut to review next week.

**Comment by andaleliz** [comment]
Yes, I think that it's probably a database info situation too...but I believe it could still be remedied on the front-end. Perhaps that's not a best practice though.

Thanks for tagging it for review!

**Comment by jenstrickland** [comment]
The CSS text-transform generally doesn't work on ALL CAPS text coming from databases, and if it does it is unreliably applied.

**Comment by andaleliz** [comment]
Yep, I agree. I was thinking perhaps a JavaScript solution or something similar could work...but I'm not a dev. That could be a totally bad idea.  I probably should not even be providing solutions but since the template prompted me to, I gave it a shot :)

**Comment by desireeturner** [comment]
Weighing in on this from the development perspective, it would be nice to fix this but there is no reliable way to do so because this is an issue with the source data. We can change it programmatically, but there are cases where it would be wrong to do so. The proper solution is to fix the source data, which comes from user-inputs and therefore would require manual intervention (which we are trying to get away from when it comes to the GIBCT data).

**Comment by andaleliz** [comment]
Thanks for sharing that! I see what you're saying.  Can you please share some examples of cases where it would be wrong to do so? I know it's common for people to refer to schools by their acronyms (e.g. UCLA) but we should be spelling out the full names in their listing. Not sure where else all caps would be correct.  

I know some GIBCT data is uploaded via a CSV, and I think that it would not be terribly difficult to correct those kinds of data sources via a formula. At least to some degree.

It would also be interesting to know if programmatic correction would fix the majority or minority of the problems.

**Comment by shawkey-daniel** [comment]
Anytime an acronym is used in the institution name, it would be incorrect to use sentence or title case. I found seven examples in the first 36 institutions I reviewed. I agree it would be better generally and possibly more correct overall, but I think it is incorrect to modify it programmatically, because it is artificially modifying the source. If this is important enough, it should be corrected at the source. The code should not make assumptions and modify official source data.

**Comment by andaleliz** [comment]
> I think it is incorrect to modify it programmatically, because it is artificially modifying the source.

Yep, I'm with you on that.  So, who can help us correct the source?  If I recall correctly, the GIBCT data source is multiple CSV files... coming from multiple people/teams?

**Comment by shawkey-daniel** [comment]
The institution data is in CSV format, but the ultimate source of the CSV is WEAMS.

**Comment by 1Copenut** [comment]
I think it would be worth reaching out to WEAMS. Data that is coming back in all caps can create challenges for scanning long strings of text, and can be mis-pronounced by screen readers. Users who might have cognitive issues can also have issue reading all caps. This may not be something that can be changed, but it's worth asking.

**Comment by andaleliz** [comment]
I totally agree, would be worth reaching out to ask and raise awareness around this issue.  Who would do that?

**Comment by amyknox** [comment]
@andaleliz as mentioned in [the related slack thread](https://dsva.slack.com/archives/CE4304QPK/p1600904028004000), the Booz team is working with Education Service on this issue for the GI Bill Comparison Tool.

**Comment by andaleliz** [comment]
@raquelou - I think you asked me about all caps recently. If that's still an issue, there's some context in this ticket.
